cmd/logs.js

```diff
@@ -28,14 +29,25 @@ function validateTimestamp(ts) {
   return moment(ts, moment.ISO_8601, true).isValid();
 }
 
-function logs(from, to, command, cb) {
+function validateNumber(number) {
+  if (number == null) return true;
+  if (number === 0 || number === '0') return false;
+  return /^\d+$/.test(number);
+}
+
+function logs(command, cb) {
   const options = init(command);
-  if (!validateTimestamp(from)) return cb(new Error('Logs \'from\' timestamp invalid (ISO-8601 required)'));
-  if (!validateTimestamp(to)) return cb(new Error('Logs \'to\' timestamp invalid (ISO-8601 required)'));
+
+  // Validate input parameters
+  if (!validateTimestamp(options.start)) return handleActionFailure(new Error('Logs \'start\' timestamp invalid (ISO-8601 expected)'), cb);
+  if (!validateTimestamp(options.end)) return handleActionFailure(new Error('Logs \'end\' timestamp invalid (ISO-8601 expected)'), cb);
+  if (!validateNumber(options.number)) return handleActionFailure(new Error('Logs \'number\' parameter invalid (non-zero integer expected)'), cb);
+  if (!validateNumber(options.page)) return handleActionFailure(new Error('Logs \'page\' parameter invalid (non-zero integer expected)'), cb);
```

Comment:
If these error messages get extracted as some constants, they could be used in the tests, too, and easily changed